store/seed/release/10001__init_seed.sql

```diff
@@ -1,3 +1,21 @@
+-- Create 
+INSERT INTO
```

Comment:
If we need to add this, then it's more appropriate to be added in init_schema (like how we add the default project id)